generated/beats/fields.ecs.yml

```diff
@@ -839,6 +839,13 @@
       description: Full file path of the library.
       example: C:\Windows\System32\kernel32.dll
       default_field: false
+    - name: pe.architecture
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: CPU architecture target for the file.
+      example: x64
```

Comment:
Do we care if we make our own values here or should we use the ones that Microsoft defined?  For example, in the sensor output `x64` but Microsoft uses the nomenclature `AMD64` from `IMAGE_FILE_MACHINE_AMD64`:

https://github.com/elastic/endpoint-dev/blob/c0cc3bf7607f5a8b539ead411b85951ddf69876d/Libraries/FeatureLib/Lib/win32/Modules.cpp#L3494-L3498

https://github.com/elastic/endpoint-dev/blob/e53cb6885130572cd552b0c68b13c10d3d81f17a/Endgame/Plugins/Processes/Lib/ProcessesTask.cpp#L1721-L1722

Reply:
My thought was to normalize it like VirusTotal does, but not entirely sure if we'd have to be strict about this.